ext/zend_test/tests/zend_weakmap.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Test internal weakmap API
+--FILE--
+<?php
+
+$id1 = new \stdClass;
+$id2 = new \stdClass;
+
+var_dump(zend_weakmap_attach($id1, 1));
+var_dump(zend_weakmap_attach($id1, 3));
+var_dump(zend_weakmap_attach($id2, 2));
+
+var_dump(zend_weakmap_dump());
+
+var_dump(zend_weakmap_remove($id2));
+var_dump(zend_weakmap_remove($id2));
+
+var_dump(zend_weakmap_dump());
```

Comment:
Might want to `unset($id1)` and dump again here, to shows the actual weak part :)